From 9624d9f09d92a6f6ec3c144ef12d476f82426205 Mon Sep 17 00:00:00 2001
From: Eugene Syromyatnikov <evgsyr@gmail.com>
Date: Sun, 19 Aug 2018 10:58:11 +0200
Subject: [PATCH 023/293] xlat: update v4l2_control_ids.in

* xlat/v4l2_control_ids.in (V4L2_CID_MPEG_VIDEO_VP8_PROFILE): Renamed
from V4L2_CID_MPEG_VIDEO_VPX_PROFILE (the latter is now a synonym),
by Linux commit v4.19-rc1~137^2~270.
(V4L2_CID_MPEG_VIDEO_VP9_PROFILE): New constant, introduced by Linux
commit v4.19-rc1~137^2~269.
---
 xlat/v4l2_control_ids.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/xlat/v4l2_control_ids.in b/xlat/v4l2_control_ids.in
index 0885944..1cdda75 100644
--- a/xlat/v4l2_control_ids.in
+++ b/xlat/v4l2_control_ids.in
@@ -155,7 +155,8 @@ V4L2_CID_MPEG_VIDEO_VPX_MIN_QP			(V4L2_CID_MPEG_BASE+507)
 V4L2_CID_MPEG_VIDEO_VPX_MAX_QP			(V4L2_CID_MPEG_BASE+508)
 V4L2_CID_MPEG_VIDEO_VPX_I_FRAME_QP		(V4L2_CID_MPEG_BASE+509)
 V4L2_CID_MPEG_VIDEO_VPX_P_FRAME_QP		(V4L2_CID_MPEG_BASE+510)
-V4L2_CID_MPEG_VIDEO_VPX_PROFILE			(V4L2_CID_MPEG_BASE+511)
+V4L2_CID_MPEG_VIDEO_VP8_PROFILE			(V4L2_CID_MPEG_BASE+511)
+V4L2_CID_MPEG_VIDEO_VP9_PROFILE			(V4L2_CID_MPEG_BASE+512)
 /* CIDs for HEVC encoding. */
 V4L2_CID_MPEG_VIDEO_HEVC_MIN_QP			(V4L2_CID_MPEG_BASE + 600)
 V4L2_CID_MPEG_VIDEO_HEVC_MAX_QP			(V4L2_CID_MPEG_BASE + 601)
-- 
1.7.12.4

